js/apps/admin-ui/src/identity-providers/add/AdvancedSettings.tsx

```diff
@@ -93,8 +98,18 @@ type AdvancedSettingsProps = { isOIDC: boolean; isSAML: boolean };
 
 export const AdvancedSettings = ({ isOIDC, isSAML }: AdvancedSettingsProps) => {
   const { t } = useTranslation("identity-providers");
-  const { control } = useFormContext();
+  const {
+    control,
+    register,
+    formState: { errors },
+  } = useFormContext<IdentityProviderRepresentation>();
   const [syncModeOpen, setSyncModeOpen] = useState(false);
+  const filteredByClaim = useWatch({
+    control,
+    name: "config.filteredByClaim",
+    defaultValue: false,
```

Comment:
Not sure, but likely this should be a string to match the actual value stored in the form.
```suggestion
    defaultValue: "false",
```

Reply:
Not sure why it worked before but you are correct, it's a string